.idea/ant.xml

```diff
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project version="5">
+<project version="4">
```

Comment:
I think the changes to this file should be reverted.

Reply:
I agree - I have reverted them

Reply:
You reverted the changes, which was good, but then somehow the version 4 came back.
Maybe your editor does this change?